products/electron.md

```diff
@@ -17,22 +17,26 @@ sortReleasesBy: releaseCycle
 releases:
   # The approximate EoL going forward will be 8 months for every release
   # but this varies a lot currently due to the cadence change.
+  - releaseCycle: "19"
+    eol: false
+    release: 2022-05-24
+    latest: "19.0.0"
   - releaseCycle: "18"
     eol: false
     release: 2022-03-29
-    latest: "18.2.4"
+    latest: "18.3.0"
   - releaseCycle: "17"
     eol: false
     release: 2022-02-01
     latest: "17.4.5"
   - releaseCycle: "16"
-    eol: false
+    eol: true
```

Comment:
Better to put a date.

Reply:
Thanks for the hint, updated with 8dfe4f7f4573a7a59e2412b33205af6eeee77971